easybuild/easyconfigs/n/netCDF/netCDF-4.5.0-intel-2018.00.eb

```diff
@@ -0,0 +1,35 @@
+name = 'netCDF'
+version = '4.5.0'
+
+homepage = 'http://www.unidata.ucar.edu/software/netcdf/'
+description = """NetCDF (network Common Data Form) is a set of software libraries 
+ and machine-independent data formats that support the creation, access, and sharing of array-oriented 
+ scientific data."""
+
+toolchain = {'name': 'intel', 'version': '2018.00'}
+toolchainopts = {'pic': True, 'usempi': True}
+
+sources = ['v%(version)s.tar.gz']
+source_urls = [
+    'https://github.com/Unidata/netcdf-c/archive/'
+]
```

Comment:
@reedts Please include a SHA256 checksum (`eb --inject-checksums` can help), and collapse `source_urls` to a single line